om/hash.go

```diff
@@ -98,34 +106,42 @@ func (r *HashRepository[T]) toExec(entity *T) (val reflect.Value, exec rueidis.L
 // Save the entity under the redis key of `{prefix}:{id}`.
 // It also uses the `redis:",ver"` field and lua script to perform optimistic locking and prevent lost update.
 func (r *HashRepository[T]) Save(ctx context.Context, entity *T) (err error) {
-	val, exec := r.toExec(entity)
+	verf, exec := r.toExec(entity)
 	str, err := hashSaveScript.Exec(ctx, r.client, exec.Keys, exec.Args).ToString()
 	if rueidis.IsRedisNil(err) {
+		if r.schema.verless {
+			return nil
+		}
```

Comment:
When will this happen? If it never happens, can we just delete it?
```suggestion
```

Reply:
Oh, can we align with the original behavior to return ARGV[2] as well?

Reply:
Absolutely. Wrote this PR quickly so I could fork and use `om` - looking at this code today I don't remember a lot of it and it looks a little dodgy to me. I'll try to reimplement a bit cleaner this week.